executor/util/util.go

```diff
@@ -23,6 +23,16 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 )
 
+const (
```

Comment:
"util" packages tend to just grow without limit...  can these constants be simply placed in executor.go or something like that?